integration-tests/manual-api/test.fake.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable */
+const { channel } = require('diagnostics_channel')
+const tracer = require('dd-trace')
+
+const testStartCh = channel('dd-trace:ci:manual:test:start')
+const testFinishCh = channel('dd-trace:ci:manual:test:finish')
+const testAddTagsCh = channel('dd-trace:ci:manual:test:addTags')
+const testSuite = __filename
+
+describe('can run tests', () => {
+  beforeEach((testName) => {
+    testStartCh.publish({ testName, testSuite })
+  })
+  afterEach((status, error) => {
+    testFinishCh.publish({ status, error })
+  })
+  test('first test will pass', () => {
+    testAddTagsCh.publish({ 'test.custom.tag': 'custom.value' })
+    assert.equal(1, 1)
+  })
+  test('second test will fail', () => {
+    assert.equal(1, 2)
+  })
+  test('async test will pass', () => {
+    return new Promise((resolve) => {
+      setTimeout(() => {
+        assert.equal(1, 1)
+        resolve()
+      }, 10)
+    })
+  })
+  test('integration test', () => {
+    // Just for testing purposes, so we don't create a custom span
+    if (!process.env.DD_CIVISIBILITY_MANUAL_API_ENABLED) {
+      return Promise.resolve()
+    }
+    const testSpan = tracer.scope().active()
+    const childSpan = tracer.startSpan('custom.span', {
+      childOf: testSpan
+    })
```

Comment:
@rochdev  I added `dd-trace:ci:manual:test:addTags` but left this bit with our current API. While this could be done via events, I don't want to prematurely expand the new manual api to be able to create arbitrary spans.

Reply:
I'm not sure I understand the reasoning. Creating spans is a completely separate event, so adding an event to add tags here would not give the ability to create arbitrary spans. If you're worried about events, which would definitely be justified since it's the first time we introduce any publicly, then I would say to simply avoid events _entirely_ at that point, at least for now.

Reply:
we might not be understanding each other. I meant I'm ok with `dd-trace:ci:manual:test:addTags`, and I added it, but I **didn't** add an extra `dd-trace:ci:manual:test:addCustomSpan`, which would have to receive, at least, `childOf`, `name` and `tags`. I think it's too flexible (and dangerous) for a first version of this API